src/Symfony/Component/Lock/Tests/Store/CombinedStoreTest.php

```diff
@@ -43,12 +44,9 @@ protected function getClockDelay()
      */
     public function getStore(): PersistingStoreInterface
     {
-        $redis = new \Predis\Client(array_combine(['host', 'port'], explode(':', getenv('REDIS_HOST')) + [1 => null]));
```

Comment:
this should be kept but the null be replaced by 6379

Reply:
@nicolas-grekas ok. I have set it to default redis port